### PR TITLE
fix: Fix the layout when greater than 2 Device Pixel Ratio is used

### PR DESCRIPTION
**Pull request by jimhunty**
### The Issue
jeremy corbyn is antisemitic figurehead say jewish leaders news the times

This was due to using React Native's stylesheet property `hairlineWidth` within a styled component. This only appeared on certain device pixel ratios which meant some of the team saw it and some of the team didn't.

This fix adds in another media query for device pixel ratio to help handle these issues.

### CODE REVIEW

File: packages/slice/templates/styles/responsive.web.js [hunk 1]
```diff
@@ -7,9 +7,13 @@ export const SliceContainer = withResponsiveStyles(View, {
     align-items: center;
     border-style: solid;
     border-bottom-color: ${colours.functional.keyline};
-    border-bottom-width: ${StyleSheet.hairlineWidth}px;
+    border-bottom-width: 1px;
     flex: 1;
     justify-content: center;
+    margin-bottom: 100px;
```
Comment (colincclark) [inline packages/slice/templates/styles/responsive.web.js]: should this be left in?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/d62f3d92b2365a3eb3d041501615c92f5d701e02/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/263b9746b7345eaba0726c6bdcd46f4173bb5ee3/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3a356e710b7fcaee3e47da527343f3864e54b13e/index.html

**Pull request merged**